assets/fighters/bandit/bandit.fighter.yaml

```diff
@@ -36,6 +36,17 @@ spritesheet:
     attacking:
       frames: [14, 17]
 
+# maybe attacks should be defined in their own files and then referenced in fighters?
+# but there is some inherent connection to the attack frames in each fighters unique spritesheet also
```

Comment:
To me it feels right in the same file.

I've modded the old Star Wars Battlefront game which used _way_ to many files with dependencies on each-other and it can be a real pain when you end up with a bunch of files that are _too_ coupled and end up feeling that they should just be one big file.

Not to say we're anywhere near that yet, but I think it's probably sensible to allow the fighter attacks to be tied closely to the fighters in the same file.

Reply:
Yeah, I think continuing like this probably makes sense.